packages/kubectl.yml

```diff
@@ -1,4 +1,4 @@
 version: 1.13.2
-upperLimit: 2.0.0
+upperLimit: 1.16.1
```

Comment:
Let's make this 1.17.0, this will include all patch releases until (exclusive) 1.17.0. If we lock it too tight we will get failures on each patch release.

Reply:
Done